pipelines/fast-test.yml

```diff
@@ -218,30 +220,30 @@ stages:
         python nni_test/nnitest/run_tests.py --config config/pr_tests.yml
       displayName: Simple integration test
 
-  - job: macos
-    pool:
-      vmImage: macOS-latest
-
-    steps:
-    - template: templates/install-dependencies.yml
-      parameters:
-        platform: macos
-
-    - script: |
-        python setup.py develop
-        echo "##vso[task.setvariable variable=PATH]${HOME}/.local/bin:${PATH}"
-      displayName: Install NNI
-
-    - script: |
-        CI=true yarn --cwd ts/nni_manager test --exclude test/core/nnimanager.test.ts
-      displayName: TypeScript unit test
-
-    - script: |
-        cd test
-        python -m pytest ut
-      displayName: Python unit test
-
-    - script: |
-        cd test
-        python nni_test/nnitest/run_tests.py --config config/pr_tests.yml
-      displayName: Simple integration test
+  #- job: macos
+  #  pool:
+  #    vmImage: macOS-latest
+
+  #  steps:
+  #  - template: templates/install-dependencies.yml
+  #    parameters:
+  #      platform: macos
+
+  #  - script: |
+  #      python setup.py develop
+  #      echo "##vso[task.setvariable variable=PATH]${HOME}/.local/bin:${PATH}"
+  #    displayName: Install NNI
+
+  #  - script: |
+  #      CI=true yarn --cwd ts/nni_manager test --exclude test/core/nnimanager.test.ts
+  #    displayName: TypeScript unit test
+
+  #  - script: |
+  #      cd test
+  #      python -m pytest ut
+  #    displayName: Python unit test
+
+  #  - script: |
+  #      cd test
+  #      python nni_test/nnitest/run_tests.py --config config/pr_tests.yml
+  #    displayName: Simple integration test
```

Comment:
why removing all the tests?

Reply:
debugging

Reply:
since you have created this pr, why not uncomment them?

Reply:
The problem only appears on azure pipeline, so I had to debug inside the PR.
It should have finished now.